v1/bank_transactions/filters.py

```diff
@@ -0,0 +1,38 @@
+from django_filters import rest_framework as filters
+from django.db.models import Q
+
+from .models.bank_transaction import BankTransaction
+
+
+class AccountNumberFilter(filters.CharFilter):
+
+    def filter(self, qs, value):
+        queryset = None
```

Comment:
We do not need this `queryset = None`

Reply:
I'll removed this.  I previously used this in a different implementation but superseded with the `filter_account_number` custom filter method.